src/psyclone/tests/psyir/transformations/profile_test.py

```diff
@@ -463,23 +463,21 @@ def test_profile_kernels_dynamo0p3():
     code = str(invoke.gen()).replace("\n", "")
 
     correct_re = ("subroutine invoke.*"
-                  "use profile_mod, only: ProfileData, ProfileStart, "
-                  "ProfileEnd.*"
-                  r"TYPE\(ProfileData\), save :: profile.*"
-                  r"TYPE\(ProfileData\), save :: profile.*"
-                  r"call ProfileStart\(\"multi_invoke_psy\", "
-                  r"\"invoke_0:testkern_code:r0\", "
-                  r"(?P<profile1>\w*)\).*"
+                  "use psy_data_mod, only: PSyDataType.*"
+                  r"TYPE\(PSyDataType\), save :: psy_data.*"
```

Comment:
Could you use the `<profile1>` and `<profile2>` groups here too?

Reply:
Yes :)